.github/workflows/release.yml

```diff
@@ -108,7 +108,7 @@ jobs:
       # See https://github.com/cygwin/cygwin-install-action#parameters
       # Note that https://github.com/egor-tensin/setup-cygwin fails to install packages.
       - name: Setup Cygwin
-        uses: cygwin/cygwin-install-action@db475590d56881c6cef7b3f96f6f3dd9532ea1f4 # master
+        uses: cygwin/cygwin-install-action@006ad0b0946ca6d0a3ea2d4437677fa767392401 # master
```

Comment:
Without `check-sig: false` ?

Reply:
http://mirrors.kernel.org/sourceware/cygwin/x86_64/

check above link.

Look, the mirrors updated on 17th-Mar, to fixed the signature missing problem.
So, I think the check-sig option can be opened by default.
The new PR https://github.com/ossrs/srs/pull/3995 just upgrade the release action to latest commit too.